samples/mstest-runner/Simple1/TestConfiguration.cs

```diff
@@ -0,0 +1 @@
+[assembly: Parallelize]
```

Comment:
It's better to use the "best parallelization mode".

```suggestion
[assembly: Parallelize(Scope = ExecutionScope.MethodLevel)]
```